OpenRA.Mods.Common/Widgets/Logic/Ingame/GameTimerLogic.cs

```diff
@@ -51,7 +52,10 @@ public GameTimerLogic(Widget widget, OrderManager orderManager, World world)
 					if (status == null && shouldShowStatus())
 						return statusText();
 
-					return WidgetUtils.FormatTime(world.WorldTick, timestep);
+					var timeLimit = tlm != null ? tlm.TimeLimit : 0;
+					return timeLimit > 0 ?
+						WidgetUtils.FormatTime(timeLimit - world.WorldTick, timestep) :
```

Comment:
Minor nit: This can be deduplicated to
```csharp
var displayTick = timeLimit > 0 ? timeLimit - world.WorldTick : world.WorldTick;
return WidgetUtils.FormatTime(displayTick, timestep);
```

Not a blocker, but would be nice if this could be fixed alongside the other blockers.